lib/datadog/tracing/contrib/pg/configuration/settings.rb

```diff
@@ -25,6 +25,11 @@ class Settings < Contrib::Configuration::Settings
               o.default false
             end
 
+            option :error_handler do |o|
+              o.type :proc
+              o.default_proc(&Tracing::SpanOperation::Events::DEFAULT_ON_ERROR)
```

Comment:
Best I can tell this is the literal constant that represented the existing default behavior, so I wanted to reference it explicitly here to maintain existing behavior.